flight/core/Dispatcher.php

```diff
@@ -46,7 +46,7 @@ public function run($name, array $params = array()) {
         }
 
         // Run requested method
-        $output = $this->execute($this->get($name), $params);
+        $output = $this->get($name)(...$params);
```

Comment:
I actually like a lot of the cleanup you did in this file. I might borrow some of the ideas for the middleware feature I'm building into Flight